proposal/press_release.md

```diff
@@ -0,0 +1,13 @@
+Jupyter releases a new Telemetry System
+
+Today the Jupyter Team announced the release of its newest feature—the Jupyter Telemetry System. This new system enables anyone deploying Jupyter products to safely collect structured “event” data from Jupyter applications. This makes it easier to secure Jupyter deployments, detect intrusion, improve user experience, scale resources effectively, and minimize costs for both administrators and users. It achieves all of this while prioritizing user privacy. The result is an improved Jupyter experience for all. 
+
+Jupyter’s telemetry system has been a long time coming. Members of the Jupyter community have been exploring various ways to collect structured events inside Jupyter applications. It’s no easy task. While event data can improve the development and deployment of Jupyter applications, it’s difficult to create an effective, safe and secure event-logging system. 
+
+This new telemetry system strikes a successful balance between effective event-logging and honoring user privacy. Transparency is a key tenant. Users are always informed when data is being collected. In most cases, users can “opt-in” and dictate which events are logged. In stricter Jupyter environments where users cannot dictate the event-logging (e.g. FEDRAMP, FERPA, HIPAA, GDPR-compliant services), users will be informed by clear and transparent messages from Jupyter’s telemetry system. 
+
+While staying user-friendly, the telemetry system is both flexible and extensible for administrators. It integrates seamlessly with Jupyter-based applications and extensions and can be extended to work with non-Jupyter applications. It is highly configurable—allowing administrators to dictate what events are collected and when. It can be configured to store code executed on Jupyter Kernels, track clicks by users inside JupyterLab, record which datasets have been opened. For highly secure, multi-user environments, this new system can be configured to track usernames or personal identifiable information (PII); it has already been tested in highly regulated settings (see NYU’s ADRF and XX). 
```

Comment:
One persona that hasn't been covered much is the Jupyter extension developer. We can say something alone the lines of _Jupyter extension developers have well defined APIs and libraries to publish events decoupled from event storage_

Reply:
You're right. I think I'll remove "for administrators" in the first sentence and add a sentence for extension developers somewhere between sentence 2 and 3.